web/css/components/modal.css

```diff
@@ -20,3 +20,41 @@
   width: 248px;
   height: 136px;
 }
+
+.modal__title {
+  font-size: 30px;
+  width: 340px;
+  font-weight: bold;
+  padding: 8px 0;
+}
+
+.modal__text {
```

Comment:
modal text ? or its a text container ?

Reply:
it's the title of the modal, isn't ok like that ?